src/site/content/en/blog/optimize-lcp/index.md

```diff
@@ -1,668 +1,458 @@
-  separate page of your site.
-- For any CSS not needed for initial rendering, use
-  [loadCSS](https://github.com/filamentgroup/loadCSS/blob/master/README.md) to
-  load files asynchronously, which leverages `rel="preload"`and `onload`.
+In both of the last two cases, the browser needs to run the script or apply the stylesheet—which  usually involves waiting for network requests to finish—before it can discover the LCP resource and could start loading it. This is never optimal.
+
+To eliminate unnecessary resource load delay, your LCP resource should _always_ be discoverable from the HTML source. In cases where the resource is only referenced from an external CSS or JavaScript file, then the LCP resource should be preloaded; for example:
 
 ```html
-<link rel="preload" href="stylesheet.css" as="style" onload="this.rel='stylesheet'">
-```
+<!-- Load the stylesheet that will reference the LCP image. -->
+<link rel="stylesheet" href="/path/to/styles.css">
 
-<figure>
-  {% Img
-    src="image/admin/2fcwrkXQRQrM8w1qyy3P.png",
-    alt="Example of LCP improvement: Before and after deferring non-critical CSS",
-    width="800",
-    height="139"
-  %}
-  <figcaption>
-    Example of LCP improvement: Before and after deferring non-critical CSS
-  </figcaption>
-</figure>
+<!-- Preload the LCP image so it starts loading with the stylesheet. -->
+<link rel="preload" as="image" href="/path/to/hero-image.webp">
+```
 
-{% Aside %}
-For more details, refer to the [Defer non-critical
-CSS](/defer-non-critical-css/) guide.
+{% Aside 'warning' %}
+On most pages, ensuring that the LCP resource starts loading at the same time as the first resource is good enough, but be aware that it is possible to construct a page where none of the resources are discovered early and all of them start loading significantly later than TTFB. So while comparing with the first resource is a good way to identify opportunities to improve, it may not be sufficient in some cases, so it's still important to measure this time relative to TTFB and ensure it remains small.
 {% endAside %}
 
-### Inline critical CSS
+#### Optimize the priority the resource is given
 
-Inline any critical-path CSS used for above-the-fold content by including it
-directly in `<head>.`
+In some cases, the LCP resource is discoverable from the HTML markup, but it _still_ doesn't start loading as early as the first resource.
 
-<figure>
-  {% Img
-    src="image/admin/m0n0JsLpH9JsNnXywSwz.png",
-    alt="Critical CSS inlined",
-    width="800", height="325"
-  %}
-  <figcaption>Critical CSS inlined</figcaption>
-</figure>
+This can happen if the browser preload scanner's priority heuristics do not recognize that the resource is important, or if it determines that other resources are more important. It can also happen if your markup instructs the browser to delay loading, for example, if you set <code>[loading="lazy"](/browser-level-image-lazy-loading/)</code> on your image tag.
```

Comment:
```suggestion
This can happen if the browser preload scanner's priority heuristics do not recognize that the resource is important, or if it determines that other resources are more important. It can also happen if your markup instructs the browser to delay loading, for example, if you set [`loading="lazy"`](/browser-level-image-lazy-loading/) on your image tag.
```

Reply:
Since you're not in a table or whatever, you don't need `<code>` elements here.

Reply:
Hmmm...that must have been an artifact of my docs to markdown plugin...